lib/active_merchant/billing/gateways/pay_gate_xml.rb

```diff
@@ -251,6 +270,13 @@ def commit(action, request)
           :authorization  => response[:tid]
         )
       end
+      def commit_capture(action, authorization, request)
```

Comment:
One question.

The first commit that runs gets the auth code that is needed in the capture request call. how do i successfully merge this 2 functions into one?